modules/configuring-resources-managed-components.adoc

```diff
@@ -10,7 +10,13 @@ You can manually adjust the resources on {productname-ocp} for the following com
 * `clairpostgres` 
 * `postgres`
 
-This feature allows users to run smaller test clusters, or to request more resources upfront in order to avoid partially degraded `Quay` pods. Limitations and requests can be set in accordance with link:https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes[Kubernetes resource units].
+This feature allows users to run smaller test clusters, or to request more resources upfront in order to avoid partially degraded `Quay` pods. Limitations and requests can be set in accordance with link:https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes[Kubernetes resource units]. 
+
+The following components should not be set lower than their minimum requirements. This can cause issues with your deployment and, in some cases, result in failure of the pod's deployment.
+
+* `quay`: Minimum of 6 GB
+* `clairpostgres`: Minimum of 200 MB
+* `postgres`: Recommended minimum of 30 GB
```

Comment:
Suggest to:
* `quay`: Minimum of 6 GB memory, 2 vCPUs
* `clair`: Recommended of 2 GB memory, 2 vCPUs
* `clairpostgres`: Minimum of 200 MB memory

Reply:
@stevsmit yes, either simply remove `postgres` or add this "Storage Minimum 30GB". 
30G is talking about Storage, pv capacity. In this section, the Configurable Resources focus on cpu & memory